www/js/controllers/vehicle-map-controller.js

```diff
@@ -2,7 +2,7 @@ angular.module('pvta.controllers').controller('VehicleMapController', function (
   var bounds = new google.maps.LatLngBounds();
 
   var mapOptions = {
-    center: bounds.getCenter(),
+    center: new google.maps.LatLng(42.386270, -72.525844),
```

Comment:
Instead of just making Haigis the map's center in the `vehicle-map-controller`, can it be app-wide?  Like maybe move MapOptions into the service, if possible?